zkp/golang/README.md

```diff
@@ -1,16 +1,30 @@
 # Zeto toolkit golang library
 
+Golang implementation of Sparse Merkle Tree that is optimized for managing an append-only tree, and is able to generate proofs to be used as private inputs to a circom-based proof generator. The project also includes an end-to-end integration test that demonstrates how to use the go-rapidsnark library to generate zkSNARK proofs with WASM based circuit runtimes.
```

Comment:
Is the sparse Merkle tree always used when you use this library? Or is it just one part of the library that is only needed for certain Zeto circuits?

Reply:
just one part of the library when using certain circuits that involve using the nullifiers. In an ongoing PR I have created a top-level folder `sparse-merkle-tree` to enclose all the merkle tree related code (all the code in `internal` and `pkg` at this point). will have future top level folders like "keys", etc.